src/en/news/blog/2023/cephalocon-2023-call-for-proposals-extended/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cephalocon 2023 Call For Proposals Extended!
+date: 2023-02-08
+author: Mike Perez
+image: "assets/cephalocon-banner.png"
+tags:
+  - events
+  - cephalocon
+---
+
+We heard your feedback on the short notice for the [Call for
+Proposals](https://events.linuxfoundation.org/cephalocon/program/cfp/) and have
+extended the deadline to **Sunday, February 19, at 11:59 pm PST**!
+
+Cephalocon is the premier yearly event that brings together the global
+community of operators, developers, and researchers for Ceph, the open source
+distributed storage system designed to provide excellent performance,
+reliability, and scalability. Join new and existing community members worldwide
+to learn more about Ceph and the project’s future from the developers writing
+the code and the operators deploying it at scale.
+
+<a class="button"
+href="https://events.linuxfoundation.org/cephalocon/program/cfp">SUBMIT YOUR
+PROPOSAL</a>
+
+## Important Dates
+
+- **CFP Closes:** Sunday, February 19 at 11:59 pm PST
+- **CFP Notifications:** Friday, February 24
+- **Schedule Announcement:** Wednesday, March 1
+- **Presentation Slide Due Date:** Wednesday, April 12
+- **Event Dates:** Monday, April 17 – Tuesday, April 18 (Developer Summit Sunday, April 16)
+
+## Sponsorship
+
+[Sponsoring Cephalocon](https://events.linuxfoundation.org/cephalocon/sponsor/)
+is a unique opportunity to gain valuable mindshare with an elite audience of
+engineers, researchers, and end-users. Building on the success of Ceph Days and
+virtual Developer Summits, Cephalocon brings together more than 400 attendees
+from across the globe to showcase Ceph’s history and its future, real-world
+applications, and of course, highlight vendor solutions. Cephalocon 2023
+promises to make for incredible community building, cross-company
+collaboration, and cutting-edge training.
+
+By sponsoring this event, you will join other prominent industry sponsors in
+supporting the growth and innovation of Ceph and its global community.
+
+Contact us at [sponsorships@ceph.foundation](mailto:sponsorships@ceph.io) to
```

Comment:
the ceph.io address is singular - sponsorship@ceph.io

Reply:
ack, nice catch!